src/file-base.js

```diff
@@ -113,7 +114,13 @@ export class File {
    * @returns {string} hash of file
    */
   async hash(hashType = 'md5', progress) {
```

Comment:
Can we also pass a flag `noCache=true` in which case it will regenerate the hash. Maybe we can have only one objject like: 
```js
options: {
algorithm: string,
progress: function,
noCache: boolean
}
```

Reply:
Using an object as param now will certainly break dependencies. We can just add an extra param for now.

Also, I think __cache__ would be a better name. So `cache=true` means use cache, `cache=false` means recompute cache. It will default to `cache=true`